agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -228,6 +228,7 @@ class backhaul_manager : public btl::transport_socket_thread {
     std::string bssid_bridge_mac;
 
     std::unique_ptr<beerocks::agent_ucc_listener> m_agent_ucc_listener;
+    std::unordered_map<std::string, beerocks_message::sVapsList> m_vaps_map;
```

Comment:
Also, please add a doxygen comment, in particular specifying what the key and value are.